scraper.py

```diff
@@ -15,6 +15,11 @@ def scrapeData(coords):
 
     for key in coords['ids']:
         result = populartimes.get_id(config.api_key, key)
+        try:
```

Comment:
When I add a wrong ID, I still get the normal error, and I get an additional error saying `logging` isn't defined.